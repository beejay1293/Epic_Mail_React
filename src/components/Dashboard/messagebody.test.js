import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Messagebody from './messagebody';
import store from '../../store';
describe('Messages', () => {
  it('should render correctly all messages', () => {
    const Messages = shallow(<Messagebody store={store}/>);
  
    expect(Messages).toMatchSnapshot();
  });
});
