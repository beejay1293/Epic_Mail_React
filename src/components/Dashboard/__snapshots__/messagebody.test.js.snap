// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages should render correctly all messages 1`] = `
<ContextProvider
  value={null}
>
  <MessageBody
    Dashboardstate="Inbox"
    body="viewMessages"
    errors={Object {}}
    getMessage={[Function]}
    inbox={Array []}
    isAuthenticated={false}
    sent={Array []}
    specificMessage={Object {}}
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
    unread={Array []}
  />
</ContextProvider>
`;
